packages/appstore/src/components/wallet-content/wallet-cfds-listing.tsx

```diff
@@ -37,33 +37,35 @@ const WalletCFDsListing = observer(() => {
 
     const is_fiat = !isCryptocurrency(currency) && currency !== 'USDT';
 
+    const cfd_title = !is_mobile && (
+        <div className='cfd-accounts__title'>
+            <Text size='sm' line_height='m' weight='bold' color='prominent'>
```

Comment:
line_height is already m by default 👍 

```suggestion
            <Text size='sm weight='bold' color='prominent'>
```